docs/build/js-client/10-marine-js.md

```diff
@@ -1,63 +1,81 @@
 # Marine JS
 
-# TODO: adapt to JS Client
+> This section is experimental as the API is not stable and probably will change.
 
-## Using Marine services in Fluence JS
+## Using Marine services in JS client
 
-Fluence JS can host Marine services with Marine JS. Currently only pure single-module services are supported.
+JS client can host Marine services with Marine JS. Currently only pure single-module services are supported.
 
-Before registering the service, the corresponding WASM file must be loaded. Fluence JS package exports three helper functions for that.
+Before registering the service, the corresponding WASM file must be loaded.
 
-**loadWasmFromFileSystem**
+Here are the examples of how to load your wasm file in browser or node
```

Comment:
this is not very clear. there needs to be a clear explanation what the js code does including loading the file and how that relates to aqua. that is, this needs to be a complete example, with complete code, on how to set up for one and more than one, say two, wams modules from js-client to aqua.